djstripe/models.py

```diff
@@ -635,7 +636,11 @@ class CurrentSubscription(TimeStampedModel):
     amount = models.DecimalField(decimal_places=2, max_digits=7)
 
     def plan_display(self):
-        return djstripe_settings.PAYMENTS_PLANS[self.plan]["name"]
+        """
+        Returns current subscription plan name
+        """
+        plan_object = Plan.objects.get(interval=self.plan, amount=self.amount)
```

Comment:
in this case, transmitted stripe plan id. I thought it would be logical to rename the variable, as originally I was confused and did not know what is stored in the plan parameter. There are many places where there is a confusion in names